resources/assets/lib/follow-user-mapping-button.tsx

```diff
@@ -43,29 +53,40 @@ export default class FollowUserMappingButton extends React.Component<Props, Stat
   }
 
   render() {
-    if (currentUser.id == null || currentUser.id === this.props.userId) {
+    if (currentUser.id === this.props.userId && !this.props.alwaysVisible) {
```

Comment:
this incorrectly shows the button when `alwaysVisible` is false and user is guest